core/src/main/protobuf/metric.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto2";
```

Comment:
Just curious, why proto2 instead of proto3?

Reply:
Basically, in our last client we used that. I am reading the new advantages and I am going to upgrade to that version.